pages/_error.tsx

```diff
@@ -0,0 +1,23 @@
+// CSS
+import styles from '../styles/Error.module.scss';
+import NavBar from '../components/NavBar';
+
+export default function ErrorPage() {
+  // Return error screen
+  return (
+    <div className={styles.canvas}>
+      <NavBar currentPage='Home' />
+      <div className={styles.page}>
+        <div className={styles.container}>
+          <h1 className={styles.heading}>
+          404 PAGE NOT FOUND
```

Comment:
looking here https://nextjs.org/docs/pages/building-your-application/routing/custom-error#more-advanced-error-page-customizing, this page can appear in more cases then the `404` can we change this to in the `404` case show this message otherwise can we just say `A client side error occured`. with a button back to the homepage.